src/Symfony/Component/ExpressionLanguage/Node/BinaryNode.php

```diff
@@ -147,6 +147,10 @@ public function evaluate($functions, $values)
             case '*':
                 return $left * $right;
             case '/':
+                if (0 == $right) {
+                    throw new \DivisionByZeroError('Division by zero');
```

Comment:
the class doesn't exist on PHP 5
we should add symfony/polyfill-php70 to the composer.json of the component:

Reply:
Done